zwave/translations/en.yaml

```diff
@@ -0,0 +1,18 @@
+---
+configuration:
+  device:
+    name: Device
+    description: The device address of your Z-Wave controller.
```

Comment:
Address sounds strange in that context. I know it comes from the docs, but still :sweat_smile: 

```suggestion
    description: The Z-Wave controller device.
```

Reply:
Lifted from current documentation

Reply:
I know, but its under a list of devices now. Probably worth updating the docs as well :)